samples/dotnet/kernel-syntax-examples/Example40_MultiChatCompletion.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.AI.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI.ChatCompletion;
+using RepoUtils;
+
+/**
+ * The following example shows how to use Semantic Kernel with Multiple Results Text Completion as streaming
+ */
+// ReSharper disable once InconsistentNaming
+public static class Example40_MultiChatCompletion
+{
+    public static async Task RunAsync()
+    {
+        await AzureOpenAIMultiChatCompletionAsync();
+        await OpenAIMultiChatCompletionAsync();
+    }
+
+    private static async Task AzureOpenAIMultiChatCompletionAsync()
+    {
+        Console.WriteLine("======== Azure OpenAI - Multiple Chat Completion ========");
+
+        IKernel kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
+        kernel.Config.AddAzureChatCompletionService(
+            Env.Var("AZURE_OPENAI_CHAT_DEPLOYMENT_NAME"),
+            Env.Var("AZURE_OPENAI_ENDPOINT"),
+            Env.Var("AZURE_OPENAI_KEY"));
+
+        IChatCompletion completion = kernel.GetService<IChatCompletion>();
```

Comment:
Ya this stinks / breaks down the kernel expectations w/ SK (shouldn't need to pull out the chat completion bit). From a high level I'd imagine services would be valuable purely for skills/connectors etc; however, once you expect an end-user to extract services from the kernel then IMO you're in rough territory / breaking boundaries between what's a service